content/sensu-go/6.2/api/health.md

```diff
@@ -86,3 +86,83 @@ output           | {{< code shell >}}
   ]
 }
 {{< /code >}}
+
+## Get health data for your agent transport
+
+The `/health` API endpoint provides HTTP GET access to health data for your Sensu agent transport via the backend websocket.
```

Comment:
```suggestion
The `/health` API endpoint provides HTTP GET access to health data for your Sensu agent transport via the backend websocket. Sensu backend /health API information is duplicated by this agent transport API endpoint as an affordance to satisfy load balancing and security requirements of some deployments.
```

Reply:
Aiming to explain that this is duplicate information and suggest why you might want one over the other.